src/nixpacks/app.rs

```diff
@@ -49,6 +89,26 @@ impl App {
         let toml_file = toml::from_str(contents.as_str())?;
         Ok(toml_file)
     }
+
+    pub fn strip_source_path(&self, abs: &str) -> Result<String> {
+        let source_str = match self.source.to_str() {
+            Some(s) => s,
+            None => return Err(anyhow::Error::msg("Failed to parse source path")),
+        };
+        let abs_path = Path::new(abs);
+
+        // Strip source path from absolute path
+        let stripped = match abs_path.strip_prefix(source_str) {
+            Ok(p) => p,
+            Err(_e) => abs_path,
+        };
+
+        // Convert path to string
+        return match stripped.to_str() {
+            Some(v) => Ok(v.to_string()),
+            None => Err(anyhow::Error::msg("Failed to convert path to string")),
```

Comment:
for future ref can use the [`bail!`](https://docs.rs/anyhow/1.0.11/anyhow/macro.bail.html) macro